### ISSUE TITLE
Bypass for https://fssquad.com/fsblog/

### ISSUE THREAD

**Issue by thatguy1943** [FastForwardTeam/FastForward#467]
### Domain

https://fssquad.com/

### Link

Any link on the site leading to https://fssquad.com/fsblog/

### Version

0.0.0

### What browsers are you seeing the problem on?

Firefox

### What OS are you seeing the problem on?

Windows

### (Optional) Anything else?

Not sure where to add this in but here is the code to bypass this

```
domainBypass("fssquad.com", () => {
	ensureDomLoaded(() => {
		ifElement("div#wpsafe-link", d => {
            safelyNavigate(d.getElementsByTagName("a")[0].onclick())
		})
	})
})
```

Also if I can get a tutorial or something on how to add in the code to the existing code, that'd be great. Please note, I am not familiar at all with github or any of the terminology used here (pull, git, fork, etc) or using CMD commands **for github** (git, clone, etc). I mean, I know but I don't **_know_** know. Kind of like interpreting another language by watching and listening to others; you get the gist of it, but you don't actually know it. Basically, I'd need a thorough guide on the process.

**Comment by BLACKHXTM0NER0** [comment]
WHERE DO U ENTER THAT CODE TO BYPASS

**Comment by ghost** [comment]
Fixed in #907.
